dev/NavigationView/NavigationView.xaml

```diff
@@ -727,7 +727,6 @@
                                 <VisualState x:Name="OnTopNavigationPrimary">
                                     <VisualState.Setters>
                                         <Setter Target="NavigationViewItemPresenter.Margin" Value="{ThemeResource TopNavigationViewItemMargin}"/>
-                                        <Setter Target="NavigationViewItemPresenter.Foreground" Value="{ThemeResource TopNavigationViewItemForeground}" />
```

Comment:
So this was set from the style below as well ? I think this line might be overwriting the TemplateBinding below.